### ISSUE TITLE
existential block is confusingly displayed

### ISSUE THREAD

**Issue by rogpeppe** [nomeata/incredible#106]
The existential block is displayed in my browser like this:

![image](https://user-images.githubusercontent.com/66491/55514380-d1029500-565f-11e9-8210-c6e1609c9c4b.png)

After some pondering, I *think* the inner part of the block should show `P(c)` and `Q`, and we're seeing some overlaid text, but that's not clear from the display, and without any additional explanatory text, it's hard to know how it's meant to work.

**Comment by michaelgwelch** [comment]
Yeah I had the same issue when I first learned the tool. Make the block bigger to work around the issue.

**Comment by nomeata** [comment]
You can expand the block; the initial width is indeed unfortunate and could be fixed.

**Comment by rogpeppe** [comment]
I don't believe it's possible to expand the block in-place, although it can be dragged onto the canvas and expanded there (although then you need to drag out some connectors to see what the function of the block is).
